src/data_import/database_reader.py

```diff
@@ -0,0 +1,54 @@
+# TODO we need some class which help with reading data from database
+
+from database import Reader
+from typing import List, Optional
+from data_import import AVAILABLE_RAM_MEMORY, SIZE_OF_VALUE
+import pandas as pd
+
+
+class DatabaseReader:
+    def __init__(self, db_name: str, coll_name: str):
+        self.error = ''
+        self.need_chunks = False
+        # TODO check size od collection, get number of records
+        size = 10
+        try:
+            self.database = Reader(db_name, coll_name)
+            # TODO get columns name of collection
+            self.columns_name = list(self.database.get_nth_chunk('', []).columns)
+            self.need_chunks = size > self.get_chunksize()
+        except Exception:
+            self.error = 'There is some problem with database. Please try again.'
```

Comment:
```suggestion
        except Exception as e:
            self.error = 'There is some problem with database. Please try again.'
            print(e)
```
probably a better way would be to show a popup with the error or at least use the logging module, but I think that printing the error to the console is good enough. it's just so we know what exactly went wrong